lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -337,16 +341,23 @@ def test_masquerade_as_specific_user_on_self_paced(self):
         content = response.content
         self.assertIn("OOGIE BLOOGIE", content)
 
+    @ddt.data(
+        'john',  # Non-unicode username
+        six.u('fôô@bar'),  # Unicode username with @, which is what ENABLE_UNICODE_USERNAME feature allows
```

Comment:
`what ENABLE_UNICODE_USERNAME feature allows` -> `what the ENABLE_UNICODE_USERNAME feature allows`

Missing `the`, my bad :see_no_evil: